src/components/DisplayArticle.jsx

```diff
@@ -1,33 +1,31 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect } from "react";
 import { useParams } from "react-router-dom";
 import { getArticles } from "../modules/getArticles";
 import { Container } from "semantic-ui-react";
+import { useDispatch, useSelector } from "react-redux";
 
 const DisplayArticle = () => {
-  const [article, setArticle] = useState();
-  const [message, setMessage] = useState();
+  const dispatch = useDispatch();
+  const viewArticle = useSelector((state) => state.specificArticle);
```

Comment:
I would always strive to use the same key/attribute name in your component.

Reply:
does that mean that ```viewArticle```  should be renamed to ```displayArticle```?

Reply:
We had a discussion in the team and decided that this naming made it more clear for us